lib/openstudio-standards/prototypes/common/objects/Prototype.hvac_systems.rb

```diff
@@ -2824,13 +2863,21 @@ def model_add_psz_vav(model,
       unitary_system.setCoolingCoil(clg_coil)
       unitary_system.setSupplementalHeatingCoil(supplemental_htg_coil)
       unitary_system.setName("#{zone.name} Unitary PSZ-VAV")
-      unitary_system.setString(2, 'SingleZoneVAV') # @todo add setControlType() method
+      # The following control strategy can lead to "Developer Error: Component sizing incomplete."
```

Comment:
that's a new one.  how often and under what conditions does this occur?

Reply:
It is mentioned in the comment below, it's the same sentence.
```
# The following control strategy can lead to "Developer Error: Component sizing incomplete."
# EnergyPlus severe (not fatal) errors if there is no heating design load
```